src/lib/components/checklist.svelte

```diff
@@ -257,6 +260,22 @@
 		padding: 1em;
 	}
 
+	details[open] summary ~ * {
+		animation: sweep 0.5s ease-in-out;
```

Comment:
Animation tip; laat UI animations nooit langer duren dan 250ms. 500ms is te traag en zorgt ervoor dat de interface sluggish aanvoelt.

Reply:
Heb de animation duration verminderd naar 0.25s.